### PR TITLE
Exercises of css 2.5.1

### PR DESCRIPTION
**Pull request by MaattiiP**
Adding exercises 2.5.1(Include the exercises 2.3)

### CODE REVIEW

File: 00-HTML-CSS-basics/exe-css/exe-2.5.1/index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  <script>window.jQuery || document.write('<script src="js/vendor/jquery-{{JQUERY_VERSION}}.min.js"><\/script>')</script>
+  <script src="js/plugins.js"></script>
+  <script src="js/main.js"></script>
+
+  <!-- Google Analytics: change UA-XXXXX-Y to be your site's ID. -->
+  <script>
+    window.ga = function () { ga.q.push(arguments) }; ga.q = []; ga.l = +new Date;
+    ga('create', 'UA-XXXXX-Y', 'auto'); ga('send', 'pageview')
+  </script>
+  <script src="https://www.google-analytics.com/analytics.js" async defer></script>
```
Comment (ruggero-nardi) [inline 00-HTML-CSS-basics/exe-css/exe-2.5.1/index.html]: I've seen this code in other pull requests too. This is importing jQuery and google analytics. Is this being used? If not can you remove it?

**Review by ruggero-nardi — CHANGES_REQUESTED** [review]
Check those scripts importing external libraries

File: 00-HTML-CSS-basics/exe-css/exe-2.5.1/index.html (hunk 1 above)
Comment (MaattiiP) [reply 00-HTML-CSS-basics/exe-css/exe-2.5.1/index.html]: I think they're not being used, so I can get them out. That is right?

Comment (ruggero-nardi) [reply 00-HTML-CSS-basics/exe-css/exe-2.5.1/index.html]: Yes, you can remove them

**Review by ruggero-nardi — APPROVED** [review]

**Pull request merged**